platforms/shared/configuration/roles/helm_lint/vars/main.yaml

```diff
@@ -39,4 +39,6 @@ charts:
   node_registration: node-initial-registration
   cli: fabric_cli
   commit_chaincode_job: commit_chaincode 
+  approve_chaincode_job: approve_chaincode
   node: node
+
```

Comment:
additional newline might be the reason of linting error in molecule, run the molecule test and fix the errors